core/config/locales/en.yml

```diff
@@ -69,6 +69,7 @@ en:
       spree/line_item:
         description: Item Description
         name: Name
+        pre_tax_amount: Pre-Tax Refund Amount
```

Comment:
This column does not exist anymore, and is not used for refunds anymore either (see #941 and #706). 

Besides, it's use further up is for a return item form, not a line item form.